### PR TITLE
Change route so home page > results

### PR DESCRIPTION
**Pull request by danielswaine**

### CODE REVIEW

File: app/views/static_pages/_public_results.html.erb [hunk 1]
```diff
@@ -5,8 +5,6 @@
 
 <div class="jumbotron center">
   <h1>Official Results <%= Time.now.year %></h1>
-  <br />
-  <p>Results will be published here once they are known.</p>
```
Comment (joecridge) [inline app/views/static_pages/_public_results.html.erb]: Can you put this back and just wrap it in an `<% if %>` statement that checks whether all challenges have been published? That way we don't need to create a separate issue to remind us to do it later. We can check for specific challenges when we come to #61

**Comment by danielswaine** [comment]
Like this? 

  <% if !challenge.published? %>
    <br />
    <p>Results will be published here once they are known.</p>
  <% end %>

**Comment by joecridge** [comment]
```erb
<% unless Challenge.all.reduce(true) { |mem, ch| mem && ch.published? } %>
  <!-- ... will be published when known ... -->
<% end %>
```

Or something to that effect.

**Comment by danielswaine** [comment]
The code above didn't work...

**Pull request merged**